dbux-code/src/dialogs/survey1.js

```diff
@@ -157,15 +157,15 @@ const survey1 = {
   nodes: {
     waitToStart: {
       kind: DialogNodeKind.Message,
-      async enter(graphState, stack, { waitAtMost }) {
+      async enter(graphState, stack, { resume, waitAtMost }) {
         const waitDelay = 1 * 60 * 60;
         const projectManager = getOrCreateProjectManager();
         const firstBug = projectManager.projects.getByName('express').getOrLoadBugs().getById(1);
         return Promise.race([
           waitForBugSolved(firstBug),
           waitAtMost(waitDelay),
           waitForBugFailedNTimes(firstBug, 3)
-        ]);
+        ]).then(() => resume());
```

Comment:
No need to expose `resume` to user.
Instead: Always take care of pending `wait` promise after correpsonding function is called during the `dialog loop`.